spec/pushmi_pullyu/preservation_queue_spec.rb

```diff
@@ -4,15 +4,18 @@
 RSpec.describe PushmiPullyu::PreservationQueue do
   describe 'a queue with 3 items in it' do
     let(:queue) { PushmiPullyu::PreservationQueue.new(poll_interval: 0, queue_name: 'test:pmpy_queue') }
+    let!(:redis) { Redis.new }
```

Comment:
I wanted to clean redis in the after block. So my options is to create an instance variable in the before block so that I can reuse in the after block (which rubocop rspec will yell at you for). Or just create a let banged and use that variable which is what I did instead. 
These `let`s are also before the before block also.